Makefile

```diff
@@ -13,14 +13,21 @@ antivirus:
 	$(eval export DC_PROFILES=${DC_PROFILES} --profile antivirus)
 	@true
 
+.PHONY: sms-provider-stub
+sms-provider-stub:
```

Comment:
I might be tempted to change the profile name to just `stubs` and then (now or eventually) also include the email stubs?

Reply:
I thought about this too and was 50/50.

I ended up going for just sms-provider-stub because

- it doesn't make people think the email stub will be set up because that isn't available at the moment
- easy to change in the future if we add the email stub
- people might want to configure just one of the stubs so might want to be able to differentiate (but probably unlikely so think this argument is fairly weak)

I think leave as is and hope the person to come add the email stubs will make the appropriate decision